generator/src/test/java/com/scottlogic/deg/generator/outputs/targets/FileOutputTargetTests.java

```diff
@@ -0,0 +1,86 @@
+package com.scottlogic.deg.generator.outputs.targets;
+
+import com.scottlogic.deg.common.profile.Profile;
+import com.scottlogic.deg.generator.outputs.formats.OutputFormat;
+import com.scottlogic.deg.generator.utils.FileUtils;
+import org.junit.jupiter.api.Test;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class FileOutputTargetTests {
+    //Create Mocks
+    private Path mockFilePath = mock(Path.class);
+    private FileUtils mockFileUtils = mock(FileUtils.class);
+    private OutputFormat mockOutputFormat = mock(OutputFormat.class);
+    private Path mockParentPath = mock(Path.class);
+
+    @Test
+    public void validate_generateOutputFileIsADirectory_throwsException(){
+        //Arrange
```

Comment:
Personally, in most cases I don't think the `//Arrange //Act //Assert` comments are necessary.  In a test like this, the blank lines between each section should be enough.

However I know some people do think you should include them religiously.